partition/androidboot.go

```diff
@@ -75,3 +76,13 @@ func (a *androidboot) SetBootVars(values map[string]string) error {
 	}
 	return env.Save()
 }
+
+func (a *androidboot) Reboot() error {
+	// Write argument so we reboot to recovery partition
+	param := []byte("recovery\n")
+	if err := ioutil.WriteFile("/run/systemd/reboot-param", param, 0644); err != nil {
```

Comment:
Re: using AtomicWrite this is an ephemeral archive, so no, it does not make much sense imo.